deb/common/docker-ce.postinst

```diff
@@ -7,6 +7,10 @@ case "$1" in
 			if ! getent group docker > /dev/null; then
 				groupadd --system docker
 			fi
+                        # TODO Needs upgrade vs. install logic handling here
+                        if ctr --namespace docker container info dockerd 2&>1  > /dev/null ; then
+                            docker engine init
+                        fi
```

Comment:
This script is **far** too naive for the final product, but should be enough to get us bootstrapped.